examples/TestDriverRN063/src/screens/TouchablesScreen.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Button, TouchableOpacity, ScrollView, Text, TextInput } from 'react-native';
+import { Button, Pressable, TouchableOpacity, ScrollView, Text, TextInput } from 'react-native';
```

Comment:
random thought - it might be worthwhile to add prettier (in the future) to this library so the codestyle is consistent with what we have in our main repo

Reply:
We actually do have prettier, though I think it's misconfigured such that it doesn't check `.tsx` files (just `.js` and `.ts`) 🤦 .